fontforge/ofl.c

```diff
@@ -34,9 +34,6 @@
 
 static char *ofl_eng[] = {
 /* sil.org license by Authors Nicolas Spalinger & Victor Gaultney, 2007feb26 */
-    "Copyright (c) %1$d, %2$s (<URL|email>),",	/* <Date> <Copyright Holder> */
-    "with Reserved Font Name %s.",		/* <Font Family Name> */
-    "",
     "This Font Software is licensed under the SIL Open Font License, Version 1.1.",
```

Comment:
I fixed this while I was here. This was another issue noted by @kenlunde (_cf._ #4529). We're already writing the dedicated OTF `name` table string for the copyright string, so this is needless duplication which bloats files.